0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,96 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection protocol
```

Comment:
It sure is (it's explained in the introduction), I just don't like it in the title of the protocol.